Applications/Calculator/CalculatorWindow.gml

```diff
@@ -0,0 +1,273 @@
+        }
+
+        @GUI::Button {
+            name: "clear_button"
+            text: "C"
+            fixed_width: 60
+            fixed_height: 28
+            foreground_color: "brown"
+        }
+    }
+
+    @GUI::Widget {
+        layout: @GUI::HorizontalBoxLayout {}
+
+        @GUI::Button {
+            name: "mem_clear_button"
+            text: "MC"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "red"
+        }
+
+        @GUI::Widget {
+            fixed_width: 5
+        }
+
+        @GUI::Button {
+            name: "7_button"
+            text: "7"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "8_button"
+            text: "8"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "9_button"
+            text: "9"
+            fixed_width: 35
+            fixed_height: 28
+        }
+
+        @GUI::Button {
+            name: "divide_button"
+            text: "/"
+            fixed_width: 35
+            fixed_height: 28
+        }
+
+        @GUI::Button {
+            name: "sqrt_button"
+            text: "sqrt"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+    }
+
+    @GUI::Widget {
+        layout: @GUI::HorizontalBoxLayout {}
+
+        @GUI::Button {
+            name: "mem_recall_button"
+            text: "MR"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "red"
+        }
+
+        @GUI::Widget {
+            fixed_width: 5
+        }
+
+        @GUI::Button {
+            name: "4_button"
+            text: "4"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "5_button"
+            text: "5"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "6_button"
+            text: "6"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "multiply_button"
+            text: "*"
+            fixed_width: 35
+            fixed_height: 28
+        }
+
+        @GUI::Button {
+            name: "mod_button"
+            text: "%"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+    }
+
+    @GUI::Widget {
+
+        layout: @GUI::HorizontalBoxLayout {}
+
+        @GUI::Button {
+            name: "mem_save_button"
+            text: "MS"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "red"
+        }
+
+         @GUI::Widget {
+            fixed_width: 5
+         }
```

Comment:
The indentation here is 3 spaces for some reason
```suggestion
         @GUI::Widget {
             fixed_width: 5
         }
```

Reply:
No, wait.
Everything is tabbed in 9 spaces. make sure all the lines have a multiple of 4 spaces as indentation.